src/test/java/test/utils/repository/standard/PersonRepository.java

```diff
@@ -189,4 +189,15 @@
 
     @Async
     ListenableFuture<List<Person>> findByLastname(String lastname);
+    
+    //distinct methods
+    public Long countDistinctPersonByFirstname(String firstname);
+    
+    public Long countPersonDistinctByFirstname(String firstname);
+    
+    public List<Person> findPersonDistinctByFirstname(String firstname);
```

Comment:
`Person` in `findPersonDistinctByFirstname` is not really needed, and it would be worth to test it as well